.buildkite/scripts/bintray_publish.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source .buildkite/scripts/shared.sh
+
+if [[ "${FAKE_RELEASE_TAG}" ]]; then
```

Comment:
I'd suggest `${FAKE_RELEASE_TAG:-}` if this isn't necessarily set, since `set -u` will fail on the unbound variable otherwise.

Reply:
Yup, fixed!